module/Company/src/Company/Controller/CompanyController.php

```diff
@@ -16,7 +16,7 @@ public function listAction()
     {
         $companyService = $this->getCompanyService();
         $featuredPackage = $companyService->getFeaturedPackage();
-        if ($featuredPackage == null) {
+        if ($featuredPackage === null) {
```

Comment:
Functionally they are exactly the same, so there should be no differences which could cause "bugs".

I'd argue this is very similar to the `ID`/`Id`-situation. We should standardize on what to use (`is_null`, even though I also used `===`) and then apply it all at once (e.g., in another PR).

Reply:
The functionality is indeed the same. Since we are using different methods throughout the code anyway, and never agreed on a common method we can leave it as is for now. If we want to swap, we should do it together with other code quality improvements I guess.